### DISCUSSION TITLE
Release Date

### DISCUSSION THREAD

**Discussion by acepm-ams** [fullstackhero/dotnet-microservices-boilerplate#63] [General]
Dear Fullstackhero team,

I hope this message finds you well. I am writing to you today to express my appreciation for the great work you are doing. I recently tried out your .NET boilerplate clean SAAS architecture and I must say it is really awesome. It has been a great help to me in my projects.

I am currently planning for the future and have a requirement for microservices architecture. While browsing through your website, I came across the link for .NET microservices boilerplate, but noticed that it is still in the "coming soon" stage. I am really interested in this and was wondering if you could provide any information about when the final version of the .NET microservices boilerplate will be released.

I am eagerly looking forward to the release of the .NET microservices boilerplate and would be grateful if you could let me know when it will be available.

Thank you for your time and keep up the great work!

**Comment by iammukeshm** [comment]
I cannot give a solid release date for now, since I work on it during my free times only. However, I can say that the project is in good shape now, and almost 60% of the work is completed. If things go as planned, maybe the complete version of this boilerplate (along with a blazor / nextjs web app) will be available in a couple of months.
